crates/swc_ecma_lints/src/rules/no_console.rs

```diff
@@ -12,44 +12,49 @@ const MESSAGE: &str = "Unexpected console statement";
 
 #[derive(Debug, Default, Clone, Serialize, Deserialize)]
 pub struct NoConsoleConfig {
-    // not used for now
     allow: Option<AHashSet<String>>,
 }
 
 pub fn no_console(
     config: &RuleConfig<NoConsoleConfig>,
     top_level_ctxt: SyntaxContext,
 ) -> Option<Box<dyn Rule>> {
-    let rule_reaction = config.get_rule_reaction();
-
-    match rule_reaction {
+    match config.get_rule_reaction() {
         LintRuleReaction::Off => None,
-        _ => Some(visitor_rule(NoConsole::new(*rule_reaction, top_level_ctxt))),
+        _ => Some(visitor_rule(NoConsole::new(config, top_level_ctxt))),
     }
 }
 
 #[derive(Debug, Default)]
 struct NoConsole {
     expected_reaction: LintRuleReaction,
     top_level_ctxt: SyntaxContext,
+    allow: Option<AHashSet<String>>,
```

Comment:
Can you use `JsWord` instead?

Reply:
This value is compared to `JsWord`, and comparison/hash of `JsWord` is super-fast.
It's already hashed on creation, so comparison does not involve `memcmp` calls.